solidity/core/test/abacusConnectionManager.test.ts

```diff
@@ -80,10 +90,19 @@ describe('AbacusConnectionManager', async () => {
       .false;
   });
 
-  it('Owner can enroll a inbox', async () => {
+  it('Owner can enroll multiple inboxes with different domain hashes', async () => {
```

Comment:
Should this be tested with the same domain but different domain hashes?

Reply:
yes, I will add a test contract

Reply:
this is actually prohibitively difficult with `constant VERSION` because it's impossible to override